ApplicationConfigProviders/Aggregator.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\ApplicationConfigProviders;
+
+/**
+ * Aggregates configuration from ApplicationConfigProviders
+ */
+class Aggregator
+{
+    /** @var ApplicationConfigProvider[] */
+    private $providers = [];
+
+    public function addProviders( array $providers )
```

Comment:
Hmm, late coding isn't always good. It'd even be missing an interface, actually.